scr

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+# Instill/VDP Helm chart'ının yükleneceği Kubernetes namespace'ini belirleyin.
+namespace="vdp"
+
+#helm repo add instill https://helm.instill.tech
+#helm repo update
+#helm install vdp instill/vdp --devel --namespace $namespace --create-namespace &>/dev/null
+
+
+helm install vdp . --devel --namespace $namespace --create-namespace --set enableITMode=true  &>/dev/null 
```

Comment:
At this point the script path is in the repo root? Should move to `charts/vdp` to make it work.

Reply:
yes, I used helm repo directly sometimes to compare it with local helm environment.